sklearn/preprocessing/tests/test_label.py

```diff
@@ -374,6 +374,9 @@ def test_multilabel_binarizer_given_classes():
     mlb = MultiLabelBinarizer(classes=[1, 3, 2])
     assert_array_equal(mlb.fit(inp).transform(inp), indicator_mat)
 
+    # ensure a ValueError is thrown if given duplicate classes
+    assert_raises(ValueError, MultiLabelBinarizer, classes=[1, 3, 2, 3])
```

Comment:
These days, we tend to use ``assert_raise_message`` to ensure that correct error message is raised.